kubernetes-client/src/test/java/io/fabric8/kubernetes/client/internal/readiness/ReadinessTest.java

```diff
@@ -0,0 +1,82 @@
+package io.fabric8.kubernetes.client.internal.readiness;
```

Comment:
please add license header using `mvn -N license:format`

Reply:
Done. Please refer to my latest commit.